<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactEdit</class>
 <widget class="QDialog" name="ContactEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>Full Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_age">
       <property name="text">
        <string>Age:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_age"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_location">
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_location"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_gender">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_gender"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_ethnicity">
       <property name="text">
        <string>Ethnicity:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_ethnicity"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_education">
       <property name="text">
        <string>Education:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_education"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_occupation">
       <property name="text">
        <string>Occupation:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_occupation"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_email">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_email"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_phone">
       <property name="text">
        <string>Phone Number:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit_phone"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_info">
       <property name="text">
        <string>Additional Info:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_info"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ContactEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ContactEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
